sklearn/ensemble/bagging.py

```diff
@@ -52,6 +53,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
     bootstrap_features = ensemble.bootstrap_features
     support_sample_weight = has_fit_parameter(ensemble.base_estimator_,
                                               "sample_weight")
+    # Logistic regression does not support sample weights with liblinear
+    if (isinstance(ensemble.base_estimator_,LogisticRegression) and 
```

Comment:
I meant instead of
```python
if sample weights are supported:
   do A
if not:
   do B
```
you can do
```python
try:
    # do stuff with sample weights
except Exception as e:
    if #e wasn't because of sample_weights:
       raise e
    # code without sample weights.
```